readme.md

```diff
@@ -1,10 +1,24 @@
 # **Beetle**
-A beetle is tough, resilient, hard to kill, and rises above everyone else on an elytra
+_A collection of semicentralized utilities for Java development_
 
-An easy to use, powerful and modular utility system meant to be the replacement and killer for ElytraForce's old aUtils system.
+**please review this readme a248**
 
+An easy to use, powerful and modular utility system meant to be the replacement and killer for ElytraForce's old aUtils system.
 Beetle has a large focus on being object oriented and scalable.
 
+# What does it provide?
+
+Beetle is a decentralized software, so multiple applications will each have their own Beetle to work with (no sharing!)
+
+- A centralized source for safe sync/async runnable execution
+- A centralized file registration and loading system
+- A centralized command management and registration system (+ super modular node commands with Branch)
+- A centralized logger with the Beetle name that simply delegates to the platform, or slf4j
+- A centralized service registry
+- A semicentralized transactional data interaction system for SQL dialects
+- A collection of decentralized utility classes for collections
+- A collection of decentralized exceptions for specific purposes
```

Comment:
You don't need to mention decentralized in the context of utility classes or exceptions

Reply:
o k